### DISCUSSION TITLE
Welcome to flutter_tinder_learn Discussions!

### DISCUSSION THREAD

**Discussion by necoffee-web** [neco3coffee/flutter_tinder_learn#33] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sadawi-ly** [comment]
I am asaad Iam20 communication engineer still a student interested in flutter and also want to develop my-self at it and looking for some people to share me their experince
